addon/ng2/blueprints/ng2/files/__path__/main.ts

```diff
@@ -1,10 +1,12 @@
-import { bootstrap } from '@angular/platform-browser-dynamic';
+import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
 import { enableProdMode } from '@angular/core';
-import { AppComponent, environment } from './app/';<% if(isMobile) { %>
+import { AppModule, environment } from './app/';<% if(isMobile) { %>
 import { APP_SHELL_RUNTIME_PROVIDERS } from '@angular/app-shell';<% } %>
 
 if (environment.production) {
   enableProdMode();
 }
 
-bootstrap(AppComponent<% if(isMobile) { %>, [ APP_SHELL_RUNTIME_PROVIDERS ]<% } %>);
+platformBrowserDynamic().bootstrapModule(AppModule<% if(isMobile) { %>, [ 
```

Comment:
Just working on that, the providers are not injected in that way anymore, so this must be removed. I need new version of `app-shell` which exports a module and then this module can be imported into AppModule.